script/deploy-canary-demo-alpha

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+set -euxo pipefail
```

Comment:
Do we need these script or can we just put them under the github action itself

Reply:
Currently all the github actions calls the scripts so i was following the same pattern but let me know if you want to remove all these scrtips and just add it inside job steps

Reply:
Previously the script is pretty long so we just dump all commands into a file since you split them all out they are not that long anymore so I think we can just put that in github action.

Reply:
@ltrung I tried but not able to run commands directly

Reply:
@ltrung Updated such that all the jobs use a single script file and added a new env variable "NAME" based upon which script deploys correct demos.